cmd/serverless/main.go

```diff
@@ -86,7 +89,11 @@ func main() {
 	}
 }
 
-func runAgent(tagger tagger.Component) {
+func runAgent(tagger tagger.Component, compressionFactory compression.Factory) {
+
+	log.Debug("ZORK Agent is running")
+	log.Debug("ZORK Repeat, agent is running")
```

Comment:
Do we need this log lines?

Reply:
No! These are temporary. I'm getting some errors in the serverless integration tests and hoping these will help me debug what is going on. I will remove once I've tracked down the error.

Reply:
It's broken since the update to go 1.23, you can ignore it !

Reply:
Ah, amazing. Debugging that was hurting my brain!

Reply:
Actually it's since the `go.work` PR, sorry I got confused 😄 
But anyway you can ignore !